src/riak_cs_wm_bucket.erl

```diff
@@ -157,6 +162,7 @@ delete_resource(RD, Ctx=#context{user=User,
         ok ->
             riak_cs_dtrace:dt_bucket_return(?MODULE, <<"bucket_delete">>,
                                                [200], [riak_cs_wm_utils:extract_name(User), Bucket]),
+            ok = riak_cs_stats:update_with_start([bucket, delete], StartTime),
```

Comment:
Why do you have `update_with_start/3` here? Isn't this caught at `riak_cs_wm_common:update_stats` ?